kiosk/src/Services/GamepadManager.ts

```diff
@@ -258,6 +265,15 @@ class GamepadManager {
         }
     }
 
+    mergeControlValues(...values: ControlValue[]): ControlValue {
+        for (const value of values) {
+            if (value === ControlValue.Down) {
+                return ControlValue.Down;
+            }
+        }
+        return ControlValue.Up;
```

Comment:
I'm a little confused. Do the left and right controls become an up control?

Reply:
This code just returns `Down` if any of the passed in values are `Down`. Otherwise it returns `Up`. It's a helper function that is useful when multiple gamepad buttons are mapped to the same control. For example, both left-stick-left and dpad-left represent the `DPadLeft` control. It either one are active, we want to represent the `DPadLeft` control as being `Down`.